.github/workflows/build-test.yml

```
@@ -32,7 +32,7 @@ jobs:
     - name: Install dependencies
       run: dotnet restore
     - name: Build
-      run: dotnet build --configuration Release --no-restore /m
+      run: dotnet build --property WarningLevel=${{ matrix.os == 'ubuntu-latest' && '1' || '0' }} --configuration Release --no-restore /m
```

According to the [documentation](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/errors-warnings#warninglevel), the default value here is 4
```suggestion
      run: dotnet build --property WarningLevel=${{ matrix.os == 'ubuntu-latest' && '4' || '0' }} --configuration Release --no-restore /m
```